content/chronograf/v1.8/guides/dashboard-template-variables.md

```diff
@@ -144,10 +193,61 @@ FROM "telegraf".."cpu"
 WHERE time > :dashboardtime:
 GROUP BY time(:interval:)
 ```
+{{% /tab-content %}}
+{{% tab-content %}}
+- [`dashboardTime`](#dashboardtime-flux)
+- [`upperDashboardTime`](#upperdashboardtime-flux)
+- [`autoInterval`](#autointerval)
+
+### dashboardTime {id="dashboardtime-flux"}
+
+The `dashboardTime` template variable is controlled by the "time" dropdown in your Chronograf dashboard.
+
+<img src="/img/chronograf/1-6-template-vars-time-dropdown.png" style="width:100%;max-width:549px;" alt="Dashboard time selector"/>
+
+If using relative time, this variable represents the time offset specified in the dropdown (-5m, -15m, -30m, etc.) and assumes time is relative to "now".
+If using absolute time defined by the date picker, `dashboardTime` is populated with lower threshold.
+
+```js
+from(bucket: "telegraf/autogen")
+  |> range(start: dashboardTime)
+  |> filter(fn: (r) => r._measurement == "cpu" and r._field == "usage_system")
+```
+
+{{% note %}}
+To use the date picker to specify a time range in the past without "now", use `dashboardTime` as the lower limit and
+[`upperDashboardTime`](#upperdashboardtime-flux) as the upper limit.
+{{% /note %}}
+
+### upperDashboardTime{id="upperdashboardtime-flux"}
+The `upperDashboardTime` template variable is defined by the upper time limit specified using the date picker.
```

Comment:
upper time limit...same as stop time? Just curious if we're using consistent verbiage (if it makes sense to do so)